Lecture_05/Homework_05(2).py

```diff
@@ -0,0 +1,7 @@
+# 2. Створити програму, яка буде друкувати в консоль “I love Python” кожні 4.2 секунди, 
+# поки її виконання не буде перервано вручну.
+import time
+x = 1
+while x > 0:
```

Comment:
To create infinite loop you can simply write `while True` without creating a variable that will never be used